Documentation/proposals/202309-controller-id.md

```diff
@@ -0,0 +1,65 @@
+# Controller-ID Proposal
+
+## Owners:
+* [danielmellado](https://github.com/danielmellado)
+
+## Related Tickets:
+* [#4281](https://github.com/prometheus-operator/prometheus-operator/issues/4281)
+* [#4498](https://github.com/prometheus-operator/prometheus-operator/pull/4498)
+
+## Other Docs:
+* n/a
+
+### Introduction
+
+This proposal aims to implement a solution to support multiple cluster-level
+Prometheus instances running concurrently without conflicting over the same CRD.
```

Comment:
(nit) CRD -> custom resources. CRD is a definition of a custom resource (e.g. the schema definition)